### DISCUSSION TITLE
Evaluation/Logging Module Integration with PyTorch Profiler?

### DISCUSSION THREAD

**Discussion by vlomonaco** [ContinualAI/avalanche#450] [Feature Request]
We should consider adding the integration of the eval and loggings module with this new PyTorch feature? https://pytorch.org/blog/introducing-pytorch-profiler-the-new-and-improved-performance-tool/

**Comment by AndreaCossu** [comment]
Yes, that would be interesting. Let's keep in mind that it needs PyTorch 1.8.1 (plus an additional package to forward results to tensorboard). I don't think it is a priority for the moment since most of the users will use older Pytorch versions. But we should definitely keeping it in consideration for the future.
